packages/fluent-editor/src/modules/custom-uploader.ts

```
@@ -1,251 +1,97 @@
+  async upload(range: Range, files: FileList | File[]) {
+    const uploads = []
+    const fails = []
+    for (const file of Array.from(files)) {
+      if (this.validateFile(file)) {
+        uploads.push(file)
+      }
+      else {
+        fails.push(file)
+      }
+    }
+    console.log('upload', uploads, fails)
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)